test/helpers.js

```diff
@@ -5,10 +5,59 @@ import ParseError from "../src/ParseError";
 import parseTree from "../src/parseTree";
 import Settings from "../src/Settings";
 
+import diff from 'jest-diff';
+import {RECEIVED_COLOR, printReceived, printExpected} from 'jest-matcher-utils';
+import {formatStackTrace, separateMessageFromStack} from 'jest-message-util';
+
+export function ConsoleWarning(message) {
+    Error.captureStackTrace(this, global.console.warn);
+    this.name = this.constructor.name;
+    this.message = message;
+}
+Object.setPrototypeOf(ConsoleWarning.prototype, Error.prototype);
+
+/**
+ * Return the first raw string if x is tagged literal. Otherwise return x.
+ */
+export const r = x => x != null && x.hasOwnProperty('raw') ? x.raw[0] : x;
```

Comment:
This function needs a better name.

Reply:
@kevinbarabash I borrowed it from python's raw string prefix. Maybe `tex`, as you suggested above?

Reply:
I wonder if people will want to use it like
```
tex`\frac{${num}}{${den}}`
```
That could be a separate PR though if people think that's worth supporting.

Reply:
@kevinbarabash Wouldn't it be same as `String.raw`?

Reply:
I supposed it would be in which case I guess people can just use `String.raw`. 😅